include/lsst/afw/formatters/Utils.h

```diff
@@ -139,7 +141,24 @@ int extractVisitId(std::shared_ptr<lsst::daf::base::PropertySet const> const& pr
 int extractCcdId(std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
 int extractAmpId(std::shared_ptr<lsst::daf::base::PropertySet const> const& properties);
 
+/**
+Format a PropertySet into a single FITS header string
+
+@param[in] prop  Properties to format
+@return a FITS header string (exactly 80 characters per "card", no line terminators)
```

Comment:
It seems to me that it's intended to be an internal function for feeding a FITS header into AST to create a `SkyWcs`.  It's declared in a header file, but that header file isn't included anywhere that a user will pick it up (unless they explicitly include it themselves).